cli/dcos-spark/submit_builder_test.go

```diff
@@ -21,6 +21,7 @@ const keytabPrefixed = "__dcos_base64__keytab"
 const keytab = "keytab"
 const sparkAuthSecret = "spark-auth-secret"
 const marathonAppId = "spark-app"
+const sparkPackages = "group.one.id:artifact-one-id:version.one,group.two.id:artifact-two-id:version.two"
```

Comment:
Since this is only used in one test please move it inside the test method. It's better to encapsulate all the test data/logic closer together.

Reply:
Moved it to the test method.